server/tsconfig.json

```diff
@@ -14,5 +14,8 @@
   },
   "include": [
     "src/**/*.ts"
+  ],
+  "exclude": [
+    "src/**/*.spec.ts"
```

Comment:
What's the reason for this?

Reply:
was thinking that in build module we should only have files that are necessary since this is mostly unit tests wanted them to be ignored. Please let me know if it does not make sense

Reply:
@triwav removed the exclude